qiskit/providers/fake_provider/backends/perth/__init__.py

```diff
@@ -0,0 +1,16 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2022.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Mock perth backend"""
```

Comment:
Just nitpicking. Since we move the modules from `qiskit.test.mock` to `qiskit.providers.fake_provider`, can we also get rid of "mock" everywhere? This is exactly the same as other current fake backends. We can either make this changes for all the new ones, or we keep the old doc string and make a separate PR to correct them all. What do you think is better?

```suggestion
"""Fake Perth device (7 qubit)."""
```

Reply:
done